src/login/translation/defaultMessages/userProfile.js

```diff
@@ -787,7 +787,7 @@ export const userData = {
   "phones.input_help_text": (
     <FormattedMessage
       id="phones.input_help_text"
-      defaultMessage={"Phone number starting with 0 or +"}
+      defaultMessage={"Phone number starting with 0 or +46"}
```

Comment:
Could this be rephrased inline with the personal data inputs (i.e. phrased as an explanation not a statement)?

Is there use in splitting these up to give the user more focused guidance?: 
* i.e. alert them when `0` at the start is missing 
* alert them separately when the `+` is missing
* in addition tell them that `+46` is the only allowed country code in case they try a different one